hub/admin/extend_user.py

```diff
@@ -66,6 +67,30 @@ class UserCreationForm(DjangoUserCreationForm):
     )
 
 
+class OrgInline(admin.StackedInline):
+    model = OrganizationUser
+    verbose_name_plural = 'Organization'
+    view_on_site = False
+    list_display = [
+        'user',
+        'organization',
+        'is_admin',
+    ]
+    raw_id_fields = ('user', 'organization')
+    readonly_fields = ('active_subscription_status',)
+
+    def active_subscription_status(self, obj):
+        if settings.STRIPE_ENABLED:
+            return obj.active_subscription_status
```

Comment:
The field is blank when the user has a subscription; it might be nice to explicitly print the string 'None' if `obj.active_subscription_status` is empty.